### DISCUSSION TITLE
curl-8.10.1/lib/curl_setup.h:129:10: fatal error: 'curl_config.h' file not found #include "curl_config.h"

### DISCUSSION THREAD

**Discussion by archurcode** [curl/curl#15633] [General]
Failed to build curl when I use the options as below:
        --with-ssl="$OPENSSL_LIB/$abi"
        --without-libpsl
        --disable-ipv6
        --disable-manual
        --disable-pop3 
        --disable-smtp 
        --disable-imap
        --disable-smb
        --disable-telnet

Who can help me

**Comment by dfandrich** [comment]
There are people here would could probably help you, but we need more
information. Platform, compiler, where you got the sources you're using, and
complete command lines that you use to configure and compile, plus the relevant
error logs would be a good start.
